contrib/win32/win32compat/w32-doexec.c

```diff
@@ -448,6 +475,8 @@ int do_exec_windows(struct ssh *ssh, Session *s, const char *command, int pty) {
 	}
 	s->pid = pid;
 
+	do_motd_windows(pipeout[1]);
```

Comment:
Regression issues when I have $env:programdata\ssh\motd file.

1) This gets called for no-pty as well. 
Unix behavior is to print MOTD only for pty sessions.
2) SFTP, SCP is broken.
3) This method can also be "ssh -t user@ip <command>", in this case also we shouldn't print the MOTD.

Please cross check the logic present in unix.